feed/feed.go

```diff
@@ -469,12 +469,12 @@ func ServiceWithDefaultFeeds(pgxStore *store.PGXStore) *Service {
 	}))
 	r.Register(Meta{
 		ID:          "con-fwa",
-		DisplayName: "🐾 FWA 2024",
-		Description: "Furry\nA feed for all things FWA! Use #fwa, #fwa24, #fwa2024, or #furryweekend to include a post in the feed.\n\nJoin the furry feeds by following @furryli.st",
+		DisplayName: "🐾 FWA 2025",
```

Comment:
Do I recall us discussing that we were gonna try and drop the year from the titles/hash tags in the descriptions to avoid needing to update these yearly?

Reply:
Yes. Some cons often use the year (or number) as part of the main hashtag, so like I believe I said during our last discussion, I think it'd be better to automate the year/number or use some regular expression for the hashtags, so we don't reduce the accuracy of the feeds.